lib/inspec/rule.rb

```diff
@@ -89,9 +89,18 @@ def title(v = nil)
       @title
     end
 
-    def desc(v = nil)
-      @desc = unindent(v) unless v.nil?
-      @desc
+    def desc(v = nil, data = nil)
+      return @descriptions[:default] if v.nil?
+      if data.nil?
+        @descriptions[:default] = unindent(v)
+      else
+        @descriptions[v.to_sym] = unindent(data)
+      end
+    end
+
+    def descriptions(desc_array = nil)
```

Comment:
What is this for? I assume it would be for:
```
control 'my_control' do
  descriptions(default: 'test', status: 'success')
  describe ...
end
```
I am not sure this is needed at all. If its for testing id make it a __descriptions. We don't want to confuse people with two formats to do the same thing.

Reply:
Oh, that was from when I was toying with the idea of `descriptions` instead of `desc`. I'll remove it.